app/controllers/review_offer.js

```diff
@@ -120,6 +120,13 @@ export default Ember.Controller.extend(AsyncTasksMixin, {
       }, ERROR_STRATEGIES.MODAL);
     },
 
+    resumeReceivingOffer() {
+      this.runTask(async () => {
+        await this.get("offerService").resumeReceiving(this.get("model"));
+        this.send("toggleOfferOptions");
```

Comment:
Probably want to set it to false instead of toggling it. (e.g what it that function is called while the options are hidden, then they'll open up for no reason and it'll look weird)